### ISSUE TITLE
"ckanext.pages.about_menu = False" does not remove About link from nav bar.

### ISSUE THREAD

**Issue by hughstimson** [ckan/ckanext-pages#82]
I'm using CKAN v2.8.2.

According to the docs, 

> This module also gives you a quick way to remove default elements from the CKAN menu and you may need todo this in order for you to have space for the new items you add. These options are:
> 
> ckanext.pages.about_menu = False
> ckanext.pages.group_menu = False
> ckanext.pages.organization_menu = False

These settings work for *group* and *organization*, but do not affect *about*.

It looks like this can be resolved in [plugin.py line 32](https://github.com/ckan/ckanext-pages/blob/06d8ef5f14087e5d9df14d982ec4d62f21ca25f6/ckanext/pages/plugin.py#L32) by changing

```
new_args = []
    for arg in args:
        if arg[0] == 'about' and not about_menu:
            continue
```

to

```
new_args = []
    for arg in args:
        if arg[0] == 'home.about' and not about_menu:
            continue
```

**Comment by metaodi** [comment]
@hughstimson nice catch! Could you create a PR for this?

Note: this extensions is designed to work on multiple versions of CKAN, so the code should check for `home.about` (new name of route) and  `about` (old name of route).

**Comment by hughstimson** [comment]
Yup, can do a PR soon.

**Comment by jonathansberry** [comment]
I've also spotted this issue.   Since no PR appears to have been opened in the past few months, I've just opened one myself. :point_up:   Thanks to @hughstimson for explaining the fix.

**Comment by hughstimson** [comment]
Thanks @jonathansberry, I started this, got confused about how to test across versions, stopped and have been feeling vaguely guilty about it for weeks. :)

**Comment by gallexme** [comment]
Any Updates?

**Comment by amercader** [comment]
This is now fixed with #83
